framework/test/unit/modules/interoperability/base_cross_chain_update_command.spec.ts

```diff
@@ -792,85 +796,149 @@ describe('BaseCrossChainUpdateCommand', () => {
 			await expect(
 				command['beforeCrossChainMessagesExecution'](executeContext, true),
 			).resolves.toEqual([[], false]);
+
+			expect(command['verifyRoutingRules']).toHaveBeenCalledWith(
+				ccm,
+				executeContext.params,
+				executeContext.chainID,
+				true,
+			); // ***
```

Comment:
why this comment? we can remove it
```suggestion
			);
```